internal/security/anchore.go

```diff
@@ -260,30 +260,28 @@ func SetupAnchoreUser(orgId uint, clusterId string) (*User, error) {
 }
 
 func RemoveAnchoreUser(orgId uint, clusterId string) {
-	if !AnchoreEnabled {
-		return
-	}
+
 	anchorUserName := fmt.Sprintf("%v-anchore-user", clusterId)
 
 	secretItem, err := secret.Store.GetByName(orgId, anchorUserName)
 	if err != nil {
-		logger.Errorf("error fetching Anchore user secret: %v", err.Error())
+		logger.Errorf("error fetching Anchore user secret: %d/%s error: %v", orgId, anchorUserName, err.Error())
```

Comment:
Is it an error that there is no anchor secret? If not it's better to log as Info "skip deleting anchor user as no secret found" or "skip deleting anchor user as already deleted"